specifyweb/specify/tree_extras.py

```diff
@@ -299,6 +299,21 @@ def merge(node, into, agent):
 
     assert False, "failed to move all referrences to merged tree node"
 
+def bulk_move(node, into, agent):
+    from . import models
+    logger.info('Bulk move preparations children %s into %s', node, into)
```

Comment:
If this logger.info is to remain, I would recommend: 

```suggestion
    logger.info('Bulk move preparations from %s to %s', node, into)
```